src/framework/theme/components/popover/helpers/adjustment.helper.ts

```diff
@@ -33,18 +40,18 @@ export class NbAdjustmentHelper {
    *
    * @return {NbPopoverPosition} calculated position.
    * */
-  static adjust(placed: ClientRect,
+  adjust(placed: ClientRect,
```

Comment:
This looks like you have problems with formatting of the **adjust** method params